treeherder/log_parser/failureline.py

```diff
@@ -94,13 +97,39 @@ def get_kwargs(failure_line):
             if key in failure_line}
 
 
+def create_failure_line(job_log, failure_line):
+    fl = FailureLine.objects.create(repository=job_log.job.repository,
+                                    job_guid=job_log.job.guid,
+                                    job_log=job_log,
+                                    **get_kwargs(failure_line))
+    if "group" in failure_line:
+        # Omit the filename before storing.
+        group_path = path.dirname(failure_line["group"])
```

Comment:
I still think using `path` here is weird; either use posixpath directly or just implement the semantics you want here.